### PR TITLE
Fixes #20599 - show html guide fixes

### PR DESCRIPTION
**Pull request by ares**
found few more things that I'd like to separate so keeping as WIP

### CODE REVIEW

**Comment by ares** [comment]
Requires https://github.com/theforeman/smart_proxy_openscap/pull/54 to work for default xccdf profiles.

**Comment by xprazak2** [comment]
I was looking into the Default profile, it turns out to be poorly named [blank value](https://github.com/theforeman/foreman_openscap/blob/master/app/helpers/policies_helper.rb#L42). I do not know why we even have this and how it is supposed to work, it shows only when scap content has more than one profile. Besides, using it for scan results in empty report.

I think we can change it to 'Choose XCCDF profile' and remove the branching in `scap_content_profile_selector`.

**Comment by ares** [comment]
Turns out blank value is default profile which is always present, just not frequently used. So it's a valid value.

File: app/models/foreman_openscap/policy.rb [hunk 1]
```diff
@@ -28,7 +29,7 @@ class Policy < ApplicationRecord
               :if => Proc.new { |policy| policy.should_validate?('Schedule') }
 
     validates :scap_content_id, presence: true, if: Proc.new { |policy| policy.should_validate?('SCAP Content') }
-    validates :scap_content_profile_id, presence: true, if: Proc.new { |policy| policy.should_validate?('SCAP Content') }
+    #validates :scap_content_profile_id, presence: true, if: Proc.new { |policy| policy.should_validate?('SCAP Content') }
```
Comment (xprazak2) [inline app/models/foreman_openscap/policy.rb]: Maybe uncomment this and fix the default profiles separately? It should turn the tests green.

**Comment by ares** [comment]
@xprazak2 updated the validation, please take another look. Also the proxy PR should be merged first.

**Comment by xprazak2** [comment]
Merging, thanks!

**Pull request merged**